pkg/cli/user.go

```diff
@@ -80,3 +82,27 @@ func (u *User) Run(ctx context.Context, c *ClientOption, fn CreateUserFunc) erro
 
 	return fn(ssoClient)
 }
+
+type RolePermission struct {
+	Roles       object.AuthorizationRoleList `json:",omitempty"`
+	Permissions []vimType.Permission         `json:",omitempty"`
```

Comment:
Doesn't look like `Permissions` is set anywhere?

Reply:
I was thinking of the "Minimal set of vSphere roles/privileges" documented here: https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/vcp-roles.html
Ideally the role created by vcpctl would contain those, ok for that to be in a separate PR though.